syntax = "proto3";

package track_search;

import "common/track.proto";

message TrackSearchRequest {
    string spotify_track_id = 1;
}

message TrackSearchResponse {
    common.Track track = 1;
}

service TrackSearchService {
    rpc SearchTrack (TrackSearchRequest) returns (TrackSearchResponse);
}